Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/dccticketing/ui/validationresult/success/DccTicketingTransactionContextData.kt

```diff
@@ -0,0 +1,78 @@
+package de.rki.coronawarnapp.dccticketing.ui.validationresult.success
+
+import de.rki.coronawarnapp.dccticketing.core.qrcode.DccTicketingQrCodeData
+import de.rki.coronawarnapp.dccticketing.core.transaction.DccTicketingResultItem
+import de.rki.coronawarnapp.dccticketing.core.transaction.DccTicketingResultToken
+import de.rki.coronawarnapp.dccticketing.core.transaction.DccTicketingTransactionContext
+
+val dccTicketingTransactionContextPassed = DccTicketingTransactionContext(
```

Comment:
You could reduce the code by just using one DccTicketingTransactionContext and change the values of result and results with copy..